dependencyLibs/Core/src/org/droidplanner/core/mission/waypoints/DoLandStart.java

```diff
@@ -0,0 +1,55 @@
+package org.droidplanner.core.mission.waypoints;
+
+import com.MAVLink.common.msg_mission_item;
+import com.MAVLink.enums.MAV_CMD;
+
+import org.droidplanner.core.helpers.coordinates.Coord2D;
+import org.droidplanner.core.helpers.coordinates.Coord3D;
+import org.droidplanner.core.mission.Mission;
+import org.droidplanner.core.mission.MissionItem;
+import org.droidplanner.core.mission.MissionItemType;
+
+import java.util.List;
+
+public class DoLandStart extends SpatialCoordItem {
```

Comment:
@ne0fhyk: The rest of files in this directory do not contain "Impl". I do not see how the proposed renaming would simplify things. Could you explain please?